src/util/create-blacklist.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2022-present Mirage Aegis
+ * 
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ * 
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ * 
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+import { Blacklist, BlacklistT } from "../schemas/blacklist";
+
+/**
+ * Creates a blacklist using the BLACKLIST_ID env
+ * in the database connected to the MONGO_DB_URI
+ */
+export const createBlacklist = async (): Promise<void> => {
+    console.log("Creating blacklist...");
+
+    const bl: BlacklistT = new Blacklist({
+        _id: process.env.BLACKLIST_ID,
+        users: new Map<string, string>()
+    });
+    await bl.save();
+
+    console.log("Blacklist created!");
+};
```

Comment:
This module has a single function that simply creates a new blacklist. It's to be called when the blacklist can't be found